doc: remove deprecated alpha1 reference of snapshot objects

humblec opened the pull request:
alpha1 version is deprecated and this commit correct the
example doc and make it GAd version

Additional note: we have already moved the artifacts to `v1` version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on docs/snap-clone.md:
```diff
@@ -172,15 +172,15 @@ Name:         rbd-pvc-snapshot
 Labels:       <none>
 Annotations:  <none>
-API Version:  snapshot.storage.k8s.io/v1alpha1
+API Version:  snapshot.storage.k8s.io/v1
```
@humblec can you also update the describe and get outputs as they are added when cephcsi was using v1alpha1 API.?

humblec replied:
@Madhu-1 I didnt get above comment, can you explain a bit more on above ?

Madhu-1 replied:
the output of `kubectl get volumesnapshot` and `kubectl describe volumesnapshot rbd-pvc-snapshot` are from the v1alpha1 CRD. I think it got changed in V1 can you please check.

humblec replied:
@Madhu-1 updated ptal.. thanks.

humblec committed:
doc: remove deprecated alpha1 reference of snapshot objects

alpha1 version is deprecated and this commit correct the
example doc and make it GAd version

Additional note: we have already moved the artifacts to v1 version.

Signed-off-by: Humble Chirammal <hchiramm@redhat.com>

Madhu-1 commented on docs/snap-clone.md:
```diff
@@ -159,41 +159,89 @@ kubectl create -f snapshot.yaml
-NAME               AGE
-rbd-pvc-snapshot   6s
+NAME               READYTOUSE   SOURCEPVC   SOURCESNAPSHOTCONTENT   RESTORESIZE   SNAPSHOTCLASS                            SNAPSHOTCONTENT                                    CREATIONTIME   AGE
+rbd-pvc-snapshot   true         rbd-pvc                             1Gi           ocs-storagecluster-rbdplugin-snapclass   snapcontent-905e6015-2403-4302-8a4e-cd3bdf63507b   78s            79s
```
`ocs-storagecluster-rbdplugin-snapclass` this is not the name in yaml files we have

Madhu-1 commented on docs/snap-clone.md:
````diff
@@ -159,41 +159,89 @@ kubectl create -f snapshot.yaml
 
 ```console
 $ kubectl get volumesnapshot
-NAME               AGE
````
please update `get volumesnapshotclass` output also

Madhu-1 commented on docs/snap-clone.md:
```diff
@@ -159,41 +159,89 @@ kubectl create -f snapshot.yaml
-Annotations:  <none>
-API Version:  snapshot.storage.k8s.io/v1alpha1
+Annotations:  snapshot.storage.kubernetes.io/pvc-access-modes=ReadWriteOnce
+              snapshot.storage.kubernetes.io/pvc-storage-class=ocs-storagecluster-ceph-rbd
```
update this annotation value

Madhu-1 commented on docs/snap-clone.md:
```diff
@@ -159,41 +159,89 @@ kubectl create -f snapshot.yaml
-  UID:               8b9b5740-29ec-11e9-8e0f-b8ca3aad030b
+    snapshot.storage.kubernetes.io/volumesnapshot-as-source-protection
+    snapshot.storage.kubernetes.io/volumesnapshot-bound-protection
+  Generation:  1
+  Managed Fields:
+    API Version:  snapshot.storage.k8s.io/v1
+    Fields Type:  FieldsV1
+    Fields V 1:
+      F : Metadata:
+        F : Annotations:
+          .:
+          F : Snapshot . Storage . Kubernetes . Io / Pvc - Access - Modes:
+          F : Snapshot . Storage . Kubernetes . Io / Pvc - Storage - Class:
+          F : Snapshot . Storage . Kubernetes . Io / Pvc - Volume - Mode:
+      F : Spec:
+        .:
+        F : Source:
+          .:
+          F : Persistent Volume Claim Name:
+        F : Volume Snapshot Class Name:
+    Manager:      Mozilla
+    Operation:    Update
+    Time:         2022-02-09T10:42:17Z
+    API Version:  snapshot.storage.k8s.io/v1
+    Fields Type:  FieldsV1
+    Fields V 1:
+      F : Metadata:
+        F : Finalizers:
+          .:
+          V :" Snapshot . Storage . Kubernetes . Io / Volumesnapshot - As - Source - Protection ":
+          V :" Snapshot . Storage . Kubernetes . Io / Volumesnapshot - Bound - Protection ":
+    Manager:      snapshot-controller
+    Operation:    Update
+    Time:         2022-02-09T10:42:17Z
+    API Version:  snapshot.storage.k8s.io/v1
+    Fields Type:  FieldsV1
+    Fields V 1:
+      F : Status:
+        .:
+        F : Bound Volume Snapshot Content Name:
```
Please remove this section, this is not in user-readable format.

Madhu-1 commented on docs/snap-clone.md:
````diff
@@ -145,6 +145,9 @@ kubectl create -f snapshotclass.yaml
 
 ```console
 $ kubectl get volumesnapshotclass
+NAME                                        DRIVER                                  DELETIONPOLICY   AGE
````
remove old name and age section

Madhu-1 commented on docs/snap-clone.md:
```diff
@@ -159,41 +162,50 @@ kubectl create -f snapshot.yaml
-Annotations:  <none>
-API Version:  snapshot.storage.k8s.io/v1alpha1
+Annotations:  snapshot.storage.kubernetes.io/pvc-access-modes=ReadWriteOnce
+              snapshot.storage.kubernetes.io/pvc-storage-class=csi-ceph-rbd
```
`csi-ceph-rbd` is not the storageclass we create in cephcsi. refer to  https://github.com/ceph/ceph-csi/blob/266e5b3b80a486303cf6bcc48040a4bdc0da8792/examples/rbd/storageclass.yaml#L5?

Madhu-1 commented on docs/snap-clone.md:
````diff
@@ -159,41 +160,50 @@ kubectl create -f snapshot.yaml
 ### Check the status of the Snapshot
 
 ```console
 $ kubectl describe volumesnapshot rbd-pvc-snapshot
````
sorry to back here again for me looks like both cephfs and rbd snapshot/clone steps are different, we don't do describe for cephfs snapshots

humblec replied:
@Madhu-1 updated this part too. ptal.. thanks

Madhu-1 replied:
lets not have describe output for both? IMO describe output is not useful

humblec replied:
@Madhu-1  Well I added describe output  by following the comment above ```steps are different..we dont do describe for cephfs snapshots...` , now is the ask to remove it ?

humblec replied:
Also having `describe` should be fine too..

Madhu-1 replied:
>@Madhu-1 Well I added describe output by following the comment above ```steps are different..we dont do describe for cephfs snapshots...` , now is the ask to remove it ?

@humblec I never asked to add it to cephfs :)

humblec replied:
@Madhu-1 so remove the describe from both  ? already gone through different iterations here, so please clarify. IMO, having describe output is fine and its consistent across RBD and CephFS here.

Madhu-1 replied:
IMO having described output is not useful. my suggestion is to remove it

humblec replied:
@Madhu-1 done, ptal.

Madhu-1 commented on docs/snap-clone.md:
````diff
@@ -84,14 +84,57 @@ volumesnapshot.snapshot.storage.k8s.io/cephfs-pvc-snapshot created
+
+```
+
+### Check the status of CephFS Snapshot
````
can we remove this one? IMO this is not needed.
